### DISCUSSION TITLE
How can I integrate the InternVL-Chat-V1.5 model into a web page without specialized hardware or API?

### DISCUSSION THREAD

**Discussion by Joshua-Franco** [community/community#121729] [Programming Help]

**Comment by ItsRaelx** [comment]
I don't think you can add this model to your website without specialized graphic card, but you can push this model to something like https://replicate.com/ where they can host it for you and give you api access.

They have some guides on how can you do that for popular programming languages etc.

**Comment by NanduBit** [comment]
You maybe can run this in an vm(azure, digital cloud) and make an api for it

**Comment by CopperEagle** [comment]
Hey @JoshuaStudios 

There is a community hosted API for this model. The link is also on the official repository of the model you screenshotted above. [Here](https://rapidapi.com/adushar1320/api/internvl-chat) is the link.  The basic plan is free but you need to pay if you need many requests per month. Huggingface itself does not offer an inference API becuase the model runs custom code for setup, which is common for Multimodal LLMs on Huggingface AFAIK (same with moondream2).

Hope this helps. 

-------------------------------------------------

*Traducción realizada con la versión gratuita del traductor DeepL*

Hay una API alojada en la comunidad para este modelo. El enlace también está en el repositorio oficial del modelo que capturaste arriba. [Aquí](https://rapidapi.com/adushar1320/api/internvl-chat) está el enlace.  El plan básico es gratuito pero tienes que pagar si necesitas muchas peticiones al mes. Huggingface en sí no ofrece una API de inferencia porque el modelo se ejecuta código personalizado para la configuración, que es común para LLM Multimodal en Huggingface AFAIK (lo mismo con moondream2).

Espero que esto ayude
